integration/setup/common.go

```diff
@@ -351,5 +370,31 @@ func startup() {
 		} else {
 			zap.S().Infof("Compat system: port %d.", p)
 		}
+
+		// Set open telemetry tracer if jaeger endpoint is provided.
+		if *jaegerEndpointF != "" {
```

Comment:
Could we have this in separate function like `startupTracer(tb testing.TB)` and call it from here? I'm refactoring this file too so it would make conflict handling easier :hugs:

Reply:
Yeah, I like this idea! 👍

Reply:
done